pagopa/io-app
# feat: [IA-807] Remove the automatic scroll to top behaviour while navigating back

## Short description
This PR is going to remove the automatic "scroll to top" behaviour in various screens all around the application. This functionality would scroll the screen back to top on every focus, causing glitches especially with the "go back" event.

⚠️ This PR depends on https://github.com/pagopa/io-app/pull/3836 and it will create some conflict if merged before.

https://user-images.githubusercontent.com/5963683/165790512-8be72152-cfcd-4d39-958e-587d986f1c90.mov

## List of changes proposed in this pull request
- Remove the scroll behaviour from the `ScreenContent.tsx` (this is the genetic component used all around the app)
- Remove the scroll behaviour from `ProfileMainScreen.tsx`

## How to test
Go into the profile section, scroll down a bit and press an option that will redirect you to another screen. Going back to the profile screen using the back button should maintain the previous scroll offset.
